packages/mdc-auto-init/index.js

```diff
@@ -41,14 +42,18 @@ export default function mdcAutoInit(root = document, warn = CONSOLE_WARN) {
     }
 
     // TODO: Should we make an eslint rule for an attachTo() static method?
-    const component = Ctor.attachTo(node);
+    component = Ctor.attachTo(node);
     Object.defineProperty(node, ctorName, {
       value: component,
       writable: false,
       enumerable: false,
       configurable: true,
     });
   }
+
+  if (component !== null) {
```

Comment:
@kfranqueiro When I submitted the issue I was not sure. The discussion in the issue makes it clear it is about having a way to tell when all components initialize, so I changed its name to be more clear.

I guess using the component to trigger the event was suggested by @yeelan0319 to keep it DRY and reuse the [emit](https://github.com/material-components/material-components-web/blob/master/packages/mdc-base/component.js#L108) method.

I also think the PR should resolve the issue, meaning "all initialized" only until other use case arises.